crunchy/cli/compare_cmd.py

```diff
@@ -11,19 +11,34 @@
 
 @click.command()
 @click.option("--first", "-f", type=click.Path(exists=True), required=True)
-@click.option("--second", "-s", type=click.Path(exists=True), required=True)
+@click.option("--second", "-s", type=click.Path(exists=True))
+@click.option(
+    "--checksum", "-c", help="If the file should be compared to a checksum directly",
+)
 @click.option(
     "--algorithm", "-a", type=click.Choice(["md5", "sha1", "sha256"]), default="sha256"
 )
 @click.option("--dry-run", is_flag=True)
-def compare(first, second, algorithm, dry_run):
-    """Compare two files by generating checksums. Fails if two files differ."""
+def compare(first, second, algorithm, checksum, dry_run):
```

Comment:
```suggestion
def compare_checksums(first, second, algorithm, checksum, dry_run):
```

Reply:
No I want the cli commands to be short! But I can expand the description

Reply:
Fair enough 😄